$NetBSD: patch-configure.ac,v 1.4 2015/11/19 11:46:33 adam Exp $

Override EGL detection.
On Darwin, build GLX only for X11 configuration.

--- configure.ac.orig	2015-07-15 23:46:36.000000000 +0000
+++ configure.ac
@@ -78,14 +78,14 @@ case $host_os in
         ;;
     darwin*)
         build_egl=no
-        build_glx=no
+        build_glx=@BUILD_GLX@
         build_wgl=no
         build_apple=yes
         has_znow=no
         EPOXY_LINK_LIBS=""
         ;;
     *)
-        build_egl=yes
+        build_egl=@BUILD_EGL@
         build_glx=yes
         build_wgl=no
         # On platforms with dlopen, we load everything dynamically and
